lib/playback/skipnext.ts

```diff
@@ -0,0 +1,34 @@
+import putSpotifyData from "@/lib/putSpotifyData";
+
+import getAccessToken from "../getAccessToken";
+
+const next = async (
+  uri: string,
+  isTrackPlaying: boolean,
+  isSkipToPrevious = false
+) => {
+  let endpoint = "https://api.spotify.com/v1/me/player/play";
+  let method = "PUT";
```

Comment:
don't use let, it's almost never needed. If you have to switch the method, you probably need to make a new function